pkg/scheduling/requirements.go

```diff
@@ -219,16 +219,38 @@ func editDistance(s, t string) int {
 	return prevRow[n-1]
 }
 
+func getSuffix(key string) string {
+	var suffix string
+
+	if strings.Contains(key, "/") {
+		splitKey := strings.Split(key, "/")
+		suffix = splitKey[1]
+	} else {
+		suffix = key
+	}
+
+	return suffix
+
+}
+
 func labelHint(r Requirements, key string, allowedUndefined sets.Set[string]) string {
+	suffix := getSuffix(key)
+
 	for wellKnown := range allowedUndefined {
 		if strings.Contains(wellKnown, key) || editDistance(key, wellKnown) < len(wellKnown)/5 {
 			return fmt.Sprintf(" (typo of %q?)", wellKnown)
 		}
+		if strings.HasSuffix(wellKnown, suffix) {
+			return fmt.Sprintf(" (typo of %q?)", wellKnown)
+		}
 	}
 	for existing := range r {
 		if strings.Contains(existing, key) || editDistance(key, existing) < len(existing)/5 {
 			return fmt.Sprintf(" (typo of %q?)", existing)
 		}
+		if strings.HasSuffix(existing, suffix) {
```

Comment:
```suggestion
		if strings.HasSuffix(existing, getSuffix(key)) {
```

Consider inlining this